include/swift/Basic/LangOptions.h

```diff
@@ -41,6 +42,9 @@ namespace swift {
     /// Language features
     ///
 
+    /// \brief User-overridable language version to compile for.
+    version::Version EffectiveLanguageVersion = version::Version::getCurrentLanguageVersion();
```

Comment:
I'm really hoping we don't need to know that at a module level, and putting it in ModuleDecl means it's not really possible to get at it from the importer.